src/main/java/com/softserve/kickscooter/kickscootervehicle/management/dto/ScooterRawDataDto.java

```diff
@@ -0,0 +1,13 @@
+package com.softserve.kickscooter.kickscootervehicle.management.dto;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class ScooterRawDataDto {
+    UUID id;
+    double latitude;
```

Comment:
The point should work here, shouldn't it?

Reply:
A real device will be able to send ScooterRawDataDto in JSON, but won't be able to do so with Point?

Reply:
Well, now simulator is hard-coded. As i know, devices send primitives separately, and Vehicle service is driver, that can understand different incoming types, and translate Point to others